### PR TITLE
Prevents failrp from missing ! and hitting ~

### PR DESCRIPTION
**Pull request by optimumtact**
## About The Pull Request
Adds the new sexy say mod verb

## Why It's Good For The Game
Sometimes you miss the ! and hit ~ and this leads to failrp because without any IC context, ~ will be OOC in ic chat.
## Changelog
:cl: oranges
add: You can now use ~ for modifying your say verb for those intimate moments
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Yamero.

**Comment by Zytolg** [comment]
Oranges, don't remove my ability to~ nooo~

**Comment by MetalClone** [comment]
so this is why you let double beds through

**Comment by OmegaDarkPotato** [comment]
bites lip, shits pants sexily

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	var/verb_whisper = "whispers"
 	var/verb_sing = "sings"
 	var/verb_yell = "yells"
+	var/verb_sexy = "whispers huskily"
```
Comment (tralezab) [inline code/game/atoms_movable.dm:27]:
```suggestion
	var/verb_sexy = "draws out the last word in their sentence as if to either delay the next thought (as if you needed to do something before continuing) or alternatively perhaps they are singing but only to the last word, in a cutesy kind of very casual way. You would never do that in a formal setting, so make sure to adminhelp if they do. Anyways, most of space station 13 environments are pretty casual ala dorms kitchen bar and other places so I think its fine."
```

**Review by tralezab — COMMENTED** [review]
nothing sexy about grammar

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (optimumtact) [reply code/game/atoms_movable.dm:27]: sus

**Comment by Mothblocks** [comment]
Lets make a config fifile of punctuations and verbs

**Comment by uwubli** [comment]
whispers huskily, "Fuu'majin!"

**Comment by esainane** [comment]
floorbraids did nothing wrong

**Comment by Sheits** [comment]
whispers huskily, "Where is cloning"

**Comment by LemonInTheDark** [comment]
This is a threat

**Comment by PowerfulBacon** [comment]
but what if I wat to sexilly ask a questio? Or maybe I wat to shout i a sexy way. We eed support for that, its vital to gameplay.

**Comment by Autisem** [comment]
I like this change. When a felind starts ~~moaning~~ screaming sexuely in the chat I don't have to actuely read the entire text, and instead can stop after reading the verb text

**Comment by Archanial** [comment]
Put me in the screencap.

**Comment by Toastgoats** [comment]
Yet more proof that the felinid metagang shadow government controls the coderbus

**Comment by optimumtact** [comment]
shadowtgcabal

**Comment by CRITAWAKETS** [comment]
we all know armhulen is all branches of government

**Comment by Mothblocks** [comment]
im closing this because my name isnt on those walls but i'll test merge this on terry until i remember to take it off

**Pull request closed without merging**